src/main/java/ReadFileAndPrintIt.java

```diff
@@ -0,0 +1,34 @@
+package main.java;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReadFileAndPrintIt {
+    public static void main(String[] args) {
+
+        String directoryPath = "/home/buga1chuk/info.coding/test";
+        String fileName = "file.txt";
+
+        File file = new File(directoryPath, fileName);
+
+        if (file.exists()) {
+            try {
+                BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
```

Comment:
You allocate the file system descriptor and do not deallocate it after the completion of the logic.